opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -40,6 +40,18 @@
 #include <opm/parser/eclipse/Deck/Deck.hpp>
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
 
+
+namespace {
+    inline
+    void silenceCompilerWarnings() {
```

Comment:
Well made comments. I do not see these for successful builds, so perhaps the proper solution is to remove this function altogether?